pkg/cmd/repo/license/view/view.go

```diff
@@ -0,0 +1,99 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type ViewOptions struct {
+	IO         *iostreams.IOStreams
+	HTTPClient func() (*http.Client, error)
+	Config     func() (gh.Config, error)
+	License    string
+}
+
+func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Command {
+	opts := &ViewOptions{
+		IO:         f.IOStreams,
+		HTTPClient: f.HttpClient,
+		Config:     f.Config,
+		License:    "",
+	}
+
+	cmd := &cobra.Command{
+		Use:   "view <license>",
+		Short: "View an available repository license template",
+		Long: heredoc.Docf(`
+		View an available repository license template.
+		
+		%[1]s<license>%[1]s is a license name or SPDX ID.
+		`, "`"),
+		Args: cmdutil.ExactArgs(1, "gh repo licese view only takes a single license argument"),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.License = args[0]
+			if runF != nil {
+				return runF(opts)
+			}
+			return viewRun(opts)
+		},
+	}
```

Comment:
**Should we have an `Examples` section?**

If users should be able to generate a license file from this command involving redirecting stdout, then we could demonstrate the various ways to invoke it. 🤷

Reply:
Added example section with an example for redirecting stdout